server/apps/distributed-app/pom.xml

```diff
@@ -501,6 +501,7 @@
                         -javaagent:"${settings.localRepository}"/org/jacoco/org.jacoco.agent/${jacoco-maven-plugin.version}/org.jacoco.agent-${jacoco-maven-plugin.version}-runtime.jar=destfile=${basedir}/target/jacoco.exec
                         -Xms512m -Xmx2048m</argLine>
                     <reuseForks>true</reuseForks>
+                    <forkCount>2</forkCount>
```

Comment:
I'm unsure the CI will support launching 2 distributed stacks at once.

Lets see

Reply:
Me too. Let's see how the CI goes.

Reply:
1st build is green. But I would let it be part of the non-consensual part so we can validate more.